package.json

```diff
@@ -8,6 +8,8 @@
   "description": "JavaScript library for panning and zooming an SVG image from the mouse, touches and programmatically.",
   "scripts": {
     "start": "gulp",
+    "lint": "tslint 'dist/svg-pan-zoom.d.ts'",
+    "pretest": "tsc",
```

Comment:
Maybe we should just extend the `test` command like `"test": "tsc && gulp test"` - it achieves the same result and will make the reasoning about tests category in future.